pyproject.toml

```diff
@@ -57,15 +57,24 @@ ignore_patterns = [
 [tool.ruff]
 # Allow lines to be as long as 80.
 line-length = 80
+indent-width = 4
+target-version = "py39"
```

Comment:
1. 4 is the default indent width anyway
2. The docs for [target-version](https://docs.astral.sh/ruff/settings/#target-version) say to use 
  ```
  [project]
  requires-python ...
  ```
  which we are already doing

```suggestion
```